python_cross_site_scripting/ThetaTau_CMT/flows.py

```
@@ -0,0 +1,242 @@
+    def current_task(self, process):
+        if process.finished is None:
+            task = process.active_tasks().first()
+            if task:
+                flow_task = task.flow_task
+                summary = "n/a"
+                if flow_task:
+                    summary = flow_task.task_title
+                task_url = frontend_url(
+                    self.request, self.get_task_url(task), back_link="here"
+                )
+                return mark_safe('<a href="{}">{}</a>'.format(task_url, summary))
```

Ruff (0.13.3)

Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)